examples/usage.ipynb

```diff
@@ -6,9 +6,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import cirq\n",
-    "from cirq_iqm import Adonis, circuit_from_qasm\n",
-    "from cirq_iqm.optimizers import simplify_circuit"
+    "try:\n",
+    "    import cirq\n",
+    "    from cirq_iqm import Adonis, circuit_from_qasm\n",
+    "    from cirq_iqm.optimizers import simplify_circuit\n",
+    "except ImportError:\n",
+    "    print('Installing missing dependencies...')\n",
+    "    !pip install --quiet cirq cirq_iqm\n",
+    "    print('Installation ready')"
```

Comment:
If cirq is not installed and thus is installed in the except clause, it won't import Adonis, circuit_from_qasm and simplify_circuit from cirq_iqm. The try/excepts should either be separated for both cirq and cirq_iqm, or you'd need to import everything again after the `!pip install`

Reply:
True! Should be hopefully fixed now :)